### ISSUE TITLE
OpenGL 4.5 with AEO-Light 2.2

### ISSUE THREAD

**Issue by SKyeLee2** [usc-imi/aeo-light#1]
Hi, I am attempting to run AEO-Light in Windows 10.  Apparently, OpenGL 4.5 will not work in Windows 10.  Any suggestions?  I am not very very tech savvy.  Also, does AEO-Light 2.2 require MCR R2013a like AEO-LIGHT 1.1 does?  Thanks, Brad

**Comment by sjohnson-imi** [comment]
AEO-Light 2 does not require the MCR, no.  What error message are you getting?

**Comment by SKyeLee2** [comment]
Error message when I attempt to add the video file to AEO-Light 2.2:
Critical failure opening new source.  GL render glBlitFrameBuffer()

I have Windows 10
AMD Radeon HD 7400M Series
Graphics Driver Version	15.201.1151.1008

**Comment by SKyeLee2** [comment]
Anyone have any ideas about my error message when attempting to add in the .mov video file?
**Critical failure opening new source. GL render glBlitFrameBuffer()** - I believe this is because OpenGL does not work with Windows 10 (at least that's what it says when I attempt to add OpenGL to my system).
Am I doing something wrong?I

**Comment by sjohnson-imi** [comment]
Sorry for the delay: we're looking into it.

OpenGL does work on Win 10, but some Googling shows that there may be individual issues that can pop up when upgrading from an older Windows release. Some generic suggestions: uninstall and reinstall your display drivers. More at http://answers.microsoft.com/en-us/windows/forum/windows_10-update/opengl-problem-in-windows-10/aff41ab9-0b6f-4317-8dcb-67ba861584c5

**Comment by SKyeLee2** [comment]
I understand.  I truly wish now I had never upgraded to Windows 10 from Windows 7.  I appreciate your reply and I will try the uninstall and re-install of my display drivers and I'll report back here.  Thanks.

**Comment by SKyeLee2** [comment]
Well, no success.  I was able to uninstall and re-install as per the instructions, but without any difference - still getting that same error message.  I even tried compatibility mode (Windows 7) for AEO-Light 2.2.  I will await further instructions from you.  Was wondering if others are experiencing this same difficulty or do they have any suggestions?

**Comment by sjohnson-imi** [comment]
We're not able to replicate the error, no. We had had some earlier issue with the glBlit call under Linux, but that has long since been resolved (before the public Beta). Yours is the only report of a GL issue so far we've had under Windows.

**Comment by SKyeLee2** [comment]
![image](https://cloud.githubusercontent.com/assets/23012243/19817964/c76b4258-9d1b-11e6-9394-0b12f2b7e304.png)

**Comment by SKyeLee2** [comment]
![image](https://cloud.githubusercontent.com/assets/23012243/19818016/0294c0b6-9d1c-11e6-8fb2-c2ccd93bc28c.png)

This is my basic information.  Initially, I had Windows 7 64-bit and installed the free Windows 10 and later did their Anniversary upgrade.

**Comment by SKyeLee2** [comment]
WOW!  Today, I tried AEO-Light 2,2 on my other computer and it worked!  It never worked on it previously, but it works now.  Funny, it was a Windows 10 upgrade, too.  I tried one of my oldest 16 mm commercials and the audio, although not perfect, was very clear and can only get better as my skill improves.  I hope it works again, tomorrow!  I guess at this time, we can close this out...I have no idea why it worked this time, but failed every time previously.  Yippee!

**Comment by ReelboxVideo** [comment]
I am having exactly the same issue on win 7. (Critical failure opening new source. GL render glBlitFrameBuffer() ) Have tried both 2.2 and 2.3 on 3 different systems with this exact same error. I have the same Universal rig as kbrambler.

**Comment by ReelboxVideo** [comment]
We have a solution. It seems that any machines that use onboard gfx had the issue. We had no issues with 2 different machines that had nvidia gfx cards with updated drivers.

**Comment by kruegerj84** [comment]
I know this is a closed thread, but I am having the same problem. (Critical failure opening new source. GL render glBlitFrameBuffer() ).

**Comment by sjohnson-imi** [comment]
The issue seems to be tied to the implementation of the OpenGL drivers on certain Windows machines (and some cases indicate that migrating from 7 to 10 may play a role as well).

**Comment by kruegerj84** [comment]
Yes, I have windows 7. I have updated drivers of nvidiagfx cards as well and that did not work. Just looking for any new suggestions. I tried 2.3 and 2.2 AEO light. I am also not super techie
